src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemv/start-rpm-template

```diff
@@ -86,7 +86,7 @@ start() {
 
 stop() {
     echo -n $"Stopping $prog: "
-    killproc -p $PIDFILE $prog
+    killproc -p $PIDFILE -d ${{kill_timeout}} $prog
```

Comment:
I couldn't find any documentation of the `-d` parameter. Do you have any resources for that?